__tests__/Unit/pages/Standup/Standup.test.tsx

```diff
@@ -0,0 +1,148 @@
+import StandUp from '@/pages/standup';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+import { Provider } from 'react-redux';
+import { store } from '@/app/store';
+import * as AuthenticatedHooks from '@/hooks/useAuthenticated';
+import * as useGetUserQueryHook from '@/app/services/userApi';
+
+describe('Standup', () => {
+    test('should be able to render standup component if user is  logged-in and authenticated', () => {
+        jest.spyOn(useGetUserQueryHook, 'useGetUserQuery').mockImplementation(
+            (): ReturnType<typeof useGetUserQueryHook.useGetUserQuery> =>
+                ({
+                    isLoading: false,
+                } as unknown as ReturnType<
+                    typeof useGetUserQueryHook.useGetUserQuery
+                >)
+        );
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(() => ({
+            isLoggedIn: true,
+            isLoading: true,
+            userData: {
+                userName: 'Pratiyush',
+                firstName: 'Pratiyush',
+                profilePicture: '',
+            },
```

Comment:
use a fixture and export this as a constant and use it here

Reply:
fixed